### ISSUE TITLE
Custom loss function

### ISSUE THREAD

**Issue by arianaqyp** [brucefan1983/GPUMD#675]
Hi,

I have a few questions:

Custom Loss Function: I wanted to see if there is a way to change the loss function. I couldn't find it in the source code, and may have missed it. Could you point me in the right direction?

Random Seed for Hyperparameter Tuning: Is there a way to specify a random seed for hyperparameter tuning to ensure reproducibility when training a potential? Currently, training a potential with the same hyperparameters yields different results each time. I assume this is because there is no set seed for random processes like weight initialization.

Thanks,

Ariana
